s3_to_redshift.go

```diff
@@ -44,9 +46,20 @@ func fatalIfErr(err error, msg string) {
 	}
 }
 
+// rounds down a dateTime to the timeGranularity
+func truncateDate(date time.Time) time.Time {
```

Comment:
nit, I'd prefer the time granularity be passed as an argument and not implicitly referred to by global variables.